x/ccv/provider/keeper/relay.go

```diff
@@ -332,11 +332,9 @@ func (k Keeper) EndBlockCCR(ctx sdk.Context) {
 		if currentTimeUint64 > ts {
```

Comment:
Hey @MSalopek- can you add a comment on `IterateInitTimeoutTimestamp` (in different file), making it clear that it is not in order?

Reply:
Can you also open an issue on the repo (put it in the v1.1 milestone) to make `IterateInitTimeoutTimestamp` go in order? That would be a little better because it would avoid uneccesary iteration.

Reply:
Can do: #561. Can't tag it as 1.1 though, maybe I'm missing some access rights?

Are we expecting loads of chains to be removed due to timeouts? If we're iterating 5-20 items this does not seem worthwhile.